src/util/imageutils.cpp

```diff
@@ -15,9 +16,10 @@ ImageDigest digestImage(const QImage& image) {
         return ImageDigest();
     }
     QCryptographicHash cryptoHash(kImageHashAlgorithm);
-    cryptoHash.addData(
+    const QByteArray data = QByteArray::fromRawData(
```

Comment:
why not QByteArrayView?

Reply:
`QByteArrayView` only exists in Qt 6.0+. Also, `QByteArray::fromRawData` does not create a copy either, so it should be fine.

Reply:
A right, I forgot this is not Qt6-only code.